app/models/packages_inventory.rb

```diff
@@ -41,6 +42,7 @@ module Actions
   belongs_to :source, polymorphic: true, touch: true
 
   after_create { PackagesInventory.emit(self.action, self) }
+  scope :for_package, ->(package_id) { where(action: PackagesInventory::ALL_INVENTORY_ACTIONS, package_id: package_id)}
```

Comment:
```suggestion
  scope :for_package, ->(package) { where(package_id: Utils.to_id(package)) }
```

- If you specify all actions, it's not needed
- used `package` instead of `package_id` to support both an ID or a Model as argument